easybuild/easyconfigs/c/carma/carma-2.01-foss-2019b.eb

```
@@ -0,0 +1,34 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Helena Gomez <helena.gomez@hpcnow.com>, Erica Bianco, Danilo Gonzalez, Jordi Blasco
+# License::     GPL-v3.0
+# #
+
+easyblock = 'Tarball'
+
+name = 'carma'
+version = '2.01'
+
+homepage = 'http://utopia.duth.gr/~glykos/Carma.html'
+description = """carma - A molecular dynamics analysis program"""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['http://utopia.duth.gr/~glykos/progs/']
+sources = ['%(name)s.tar.gz']
+checksums = ['f32cfc2e348691f252d3593e281303d1202e31befba14565aba022331d50b9f3']
+
+preinstall_cmd = 'cd %(builddir)s/%(name)s-%(version)s/src && '
+preinstall_cmd += '$CC $CFLAGS -O -c Color_carma.c && '
+preinstall_cmd += '$FC $FFLAGS $LIBBLAS -O Color_carma.o -o carma &&'
+preinstall_cmd += 'cp -p carma ../bin/'
```

In this case I recommend to use the `CmdCp` easyblock, which provides some more structure to perform the actions in the `preinstall_cmd`:
```suggestion
start_dir = 'src'

cmds_map = [('.*', "$CC $CFLAGS -O -c Color_carma.c && $FC $FFLAGS $LIBBLAS -O Color_carma.o -o carma")]
files_to_copy = [(['carma'], 'bin')]
```